### PR TITLE
General docs housekeeping

### PR DESCRIPTION
**Pull request by aisconnolly**

### CODE REVIEW

File: docs/developer-docs/integrations/ledger/deploy-new-token.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This tutorial will guide you step-by-step to deploy your own token to the IC and
 
 1.  Ensure you have the ledger image, the private ledger interface, and the public ledger interface. If you do not have them, follow the steps in [Setup Ledger locally](./ledger-local-setup).
 
-2.  Ensure you use a recent version of DFX. If you don’t have DFX installed, follow instructions on <https://smartcontracts.org/> to install it.
+2.  Ensure you use a recent version of DFX. If you don’t have DFX installed, follow instructions on the [build](../../build/install-upgrade-remove) section to install it.
```
Comment (roman-kashitsyn) [inline docs/developer-docs/integrations/ledger/deploy-new-token.md:9]: I'm not sure "the [build] section" is correct here; I'd call it "the [install section]" or "the [Installing the SDK] section."

File: docs/developer-docs/integrations/ledger/deploy-new-token.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ This tutorial will guide you step-by-step to deploy your own token to the IC and
 
     -   the `MINT_ACC` is the account of the Principal responsible for minting and burning tokens (see the [Ledger documentation](./index.md))
 
-    -   the `ARCHIVE_CONTROLLER` is the [controller Principal](https://smartcontracts.org/docs/developers-guide/cycles-wallet.html#_controller_and_custodian_roles) of the archive canisters
+    -   the `ARCHIVE_CONTROLLER` is the [controller Principal](../../build/project-setup/cycles-wallet#_controller_and_custodian_roles) of the archive canisters
```
Comment (roman-kashitsyn) [inline docs/developer-docs/integrations/ledger/deploy-new-token.md:65]:
```suggestion
    -   the `ARCHIVE_CONTROLLER` is the [controller Principal](../../build/project-setup/cycles-wallet#controller-and-custodian-roles) of the archive canisters
```

File: docs/developer-docs/integrations/ledger/ledger-local-setup.md [hunk 3]
```diff
@@ -19,9 +19,9 @@ Follow the steps below to deploy your copy of the ledger canister to a local rep
 
     :::
 
-2.  Make sure you use a recent version of DFX. If you don’t have DFX installed, follow instructions on <https://smartcontracts.org/> to install it.
+2.  Make sure you use a recent version of DFX. If you don’t have DFX installed, follow instructions on the [build](../../build/install-upgrade-remove) section to install it.
```
Comment (roman-kashitsyn) [inline docs/developer-docs/integrations/ledger/ledger-local-setup.md:22]: Same as above: not sure the section name is correct.

**Review by roman-kashitsyn — APPROVED** [review]
FI-related changes look great; thanks a lot!

File: docs/developer-docs/integrations/ledger/deploy-new-token.md (hunk 1 above)
Comment (aisconnolly) [reply docs/developer-docs/integrations/ledger/deploy-new-token.md]: totally agree - will change.

File: docs/developer-docs/integrations/ledger/ledger-local-setup.md (hunk 3 above)
Comment (aisconnolly) [reply docs/developer-docs/integrations/ledger/ledger-local-setup.md]: changed

**Review by harpa12 — APPROVED** [review]

**Pull request merged**